app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -560,6 +566,29 @@ fun MapScreen(
                                 return@KeyboardActions
                               }
 
+                              // Easter Egg 4 : Teleport
+                              val matchTeleport =
+                                  teleportRegex.matchEntire(searchQuery.value.trim())
+                              if (matchTeleport != null) {
+                                searchQuery.value = ""
+                                val availableScreens =
+                                    listOf(
+                                        Screen.LIST,
+                                        Screen.VIEW_PROFILE,
+                                        Screen.GAMBLING,
+                                        Screen.RACK_INFO,
+                                        Screen
+                                            .ADMIN, // this one's hilarious, imagine you're not
+                                                    // admin
```

Comment:
😂😂🤣🤣

Reply:
Hope the guy who spent 5 weeks making a secure report system doesn't see this!